src/python/aqueduct_executor/operators/utils/enums.py

```diff
@@ -81,3 +81,8 @@ class SerializationType(str, Enum, metaclass=MetaEnum):
     STRING = "string"
     BYTES = "bytes"
     TF_KERAS = "tensorflow-keras-model"
+
+
+class PrintColorType(str, Enum, metaclass=MetaEnum):
+    GREEN = "\33[32m"
```

Comment:
wth is this format lol. Can you add a quick description about what this format is, so someone can recreate the colors themselves?